lib/Setup Thermostats.js

```diff
@@ -153,6 +154,7 @@ exports.setupThermostat = function (that, services)
 			.updateOnHubEvents(that.currentAccessory.id, "heatingSetpoint")	
 			.on('set', function(newHomekitValue, callback, context)
 				{
+					if(TargetState.value !== 3) return callback(null);

```

Comment:
I think this and the next change are the only ones that really matter to get the effect you are looking for.